install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2460,6 +2460,11 @@ preflight:
   #     cpu: 100m
   #     memory: 512Mi
 
+  readinessProbe:
+    # -- The initialDelaySeconds field tells the kubelet that it should wait 5 seconds before performing the first probe
```

```suggestion
    # -- For how long kubelet should wait before performing the first probe
```
"5 seconds" shouldn't be hardcoded in the doc comment.

---

Done